assets/scss/optimized-checkout.scss

```diff
@@ -1,103 +1,168 @@
+    }
+    &:active {
+        color: stencilColor("optimizedCheckout_button--primary-colorActive");
+        background-color: stencilColor("optimizedCheckout_button--primary-bgActive");
+        border-color: stencilColor("optimizedCheckout_button--primary-borderColorActive");
+    }
+}
+
+//
+// Secondary Button
+// Background & Border & Text Color & Typography
+// -----------------------------------------------------------------------------
+
+.optimizedCheckout_secondary--button {
+    color: stencilColor("optimizedCheckout_button--secondary-color");
+    background-color: stencilColor("optimizedCheckout_button--secondary-bg");
+    border-color: stencilColor("optimizedCheckout_button--secondary-borderColor");
+    font-family: stencilFontFamily("optimizedCheckout_button--secondary-font"), Arial, Helvetica, sans-serif;
+    font-weight: stencilFontWeight("optimizedCheckout_button--secondary-font");
 }
 
+//
 // Links
+// Text Color & Typography
+// -----------------------------------------------------------------------------
+
 a {
-    color: stencilColor("optimizedCheckout_links_color");
+    color: stencilColor("optimizedCheckout_link-color");
+    font-family: stencilFontFamily("optimizedCheckout_link-font"), Arial, Helvetica, sans-serif;
+    font-weight: stencilFontWeight("optimizedCheckout_link-font");
 }
 
-// Buttons
-.button {
-    background-color: stencilColor("optimizedCheckout_buttons_backgroundColor");
+//
+// Order Summary (Mobile Drawer & Modal)
+// Background
+// -----------------------------------------------------------------------------
+
+.optimizedCheckout_orderSummary {
+    background-color: stencilColor("optimizedCheckout_cart-bg");
 }
 
-// Errors
-.form-field--error {
+//
+// Order Steps
+// Background & Text Color
+// -----------------------------------------------------------------------------
 
-    .form-input,
-    .form-select,
-    .form-checkbox + .form-label::before,
-    .form-radio + .form-label::before,
-    .form-prefixPostfix-label {
-        border-color: stencilColor("optimizedCheckout_errors_color");
-    }
+.optimizedCheckout_step {
+    background-color: stencilColor("optimizedCheckout_step-bg");
 
-    .form-field-error,
-    .form-inlineMessage {
-        color: stencilColor("optimizedCheckout_errors_color");
+    &::before {
+        color: stencilColor("optimizedCheckout_step-text");
     }
+}
 
-    // Depth need to style svg with validation state
-    // scss-lint:disable SelectorDepth
-    .form-input-indicator > svg {
-        fill: stencilColor("optimizedCheckout_errors_color");
-    }
-    // scss-lint:enable SelectorDepth
+//
+// Form Errors
+// Border & Text Color
+// -----------------------------------------------------------------------------
+
+.optimizedCheckout_form-error {
+    border-color: stencilColor("optimizedCheckout_error-color");
+    color: stencilColor("optimizedCheckout_error-color");
```

Comment:
Should match with the thing you're modifying, i.e.: `stencilColor("optimizedCheckout_form-error")`